packages/manager/modules/pci/src/projects/project/users/download-openrc/download-openrc.controller.js

```diff
@@ -67,4 +68,9 @@ export default class PciUsersDownloadOpenRcController {
         this.isLoading = false;
       });
   }
+
+  getOpenRcApiVersion() {
```

Comment:
Should be managed by API, no ? :(

Reply:
Ideally it should be, but currently they dont have an API for this yet. So doing this based on `REGEX` on regions. As long as the region codes are in same format, this will work :)